lib/types.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+export type Case = 'AVERAGE' | 'BEST_CASE' | 'MEDIAN' | 'WORST_CASE'
+
+export type Coords = {
+  x: number,
+  y: number,
+  z: number
+}
+
+export type Feature = {
+  type: string,
+  geometry: {
+    type: 'string',
+    coordinates: number[][]
+  }
+}
+
+export type Grid = {
+  contains: Function,
```

Comment:
[The Function type is unsafe and should be avoided](https://flow.org/en/docs/types/functions/#toc-function-type).